STREAMS-1569 - Database guidelines (#57)
Axway/streams-open-docs: content/en/docs/install/_index.md

Not recommended for production.
{{< /alert >}}

##### Embedded MariaDB tuning

> These values of the embedded MariaDB configuration can be updated:

Reviewer: ```suggestion
The following embedded MariaDB configuration values can be updated:
```

> * The `wait-timeout` variable can be updated by setting the [Helm parameters](#helm-parameters) `embeddedMariadb.waitTimeout`.

Reviewer: ```suggestion
* `wait-timeout` - update by setting the [Helm parameters](#helm-parameters) `embeddedMariadb.waitTimeout`.
```

> * The `max-connections` variable can be updated by setting the [Helm parameters](#helm-parameters) `embeddedMariadb.maxConnections`.

Reviewer: ```suggestion
* `max-connections` - updated by setting the [Helm parameters](#helm-parameters) `embeddedMariadb.maxConnections`.
```

> {{< alert title="Note" >}}Please, refer to the [MariaDB considerations](/docs/architecture#mariadb-considerations) documentation for further details.{{< /alert >}}

Reviewer: ```suggestion
{{< alert title="Note" >}}Refer to the [MariaDB considerations](/docs/architecture#mariadb-considerations) documentation for further details.{{< /alert >}}
```

### Ingress TLS settings

> SSL/TLS is enabled by default on the embedded Ingress controller. If you don't provide any certificate, SSL will be enabled thanks to a NGINX embedded fake SSL certificate.

Reviewer: ```suggestion
SSL/TLS is enabled by default on the embedded Ingress controller. If you don't provide a certificate, SSL will be enabled with a NGINX embedded fake SSL certificate.
```
